### ISSUE TITLE
False negative in BadModuleAttributeUseLinter with nested attributes

### ISSUE THREAD

**Issue by mschwager** [duo-labs/dlint#30]
Looking for `foo.bar` in the following:

```python
import foo           
foo.bar(var).baz()
```

Should work, but since we're not recursing deeper with `generic_visit` it doesn't find the nested attribute.
